pkg/daemon/osrelease/osrelease.go

```diff
@@ -0,0 +1,151 @@
+package osrelease
+
+import (
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/ashcrow/osrelease"
```

Comment:
This is fine as is, but perhaps it'd make sense at some point to have this functionality move into that upstream library.  We could also move it to e.g. github.com/coreos/osrelease too for more shared maintenance.

Reply:
Yeah, that would be nice. I have a few ideas around some additional things we could port there such as handling paths, etc.